addons/account/models/account_move.py

```diff
@@ -1405,7 +1405,7 @@ def _compute_duplicated_ref_ids(self):
         for move in self:
             move.duplicated_ref_ids = move_to_duplicate_move.get(move, self.env['account.move'])
 
-    def _fetch_duplicate_supplier_reference(self, only_posted=False):
+    def _fetch_duplicate_supplier_reference(self, only_posted=False, only_same_invoice_date=False):
```

Comment:
default (current) behavior should actually be `True` here, especially if you don't add the parameter on line 1404